docs/src/grids.md

```diff
@@ -161,27 +173,35 @@ nothing # hide
 ```
 
 The grid contains the wavenumbers (both for real-value functions `grid.kr` and 
-for complex-valued functions `grid.k`). We populate array `∂ₓuh` is with ``\mathrm{i} k \hat{u}``:
+for complex-valued functions `grid.k`). We populate the array `∂ₓuh` with ``\mathrm{i} k \hat{u}``:
 
 ```@example 1
 @. ∂ₓuh = im * grid.kr * uh
 
 nothing # hide
 ```
 
-Then the derivative in physical space, `∂ₓu`, is obtained with an inverse Fourier 
-transform. The latter is obtained again using the `FFTW` plans but now via `ldiv!`:
+Then the derivative in physical space, `∂ₓu`, is obtained through an inverse Fourier 
+transform. For this again we use the `FFTW` plans but now via `ldiv!`:
 
 ```@example 1
 using LinearAlgebra: ldiv!
 
 ldiv!(∂ₓu, grid.rfftplan, ∂ₓuh)
+```
+
+We can plot `∂ₓu` and compare it with the analytic derivative for comparison. 
```

Comment:
Probably don't need to say "...for comparison" at the end of this sentence as you have already said "...compare it with..."?